zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -99,8 +99,13 @@ public static void main(String[] args) throws Exception {
     jettyServer.setHandler(contexts);
 
     LOG.info("Start zeppelin server");
-    jettyServer.start();
-    LOG.info("Started");
+    try {
+      jettyServer.start();
+    } catch (Exception e) {
+      LOG.error("Error while running jettyServer", e);
```

Comment:
@felixcheung I have a question. Could you tell me the benefit to use e.printStackTrace instead of using logger? I think the result is same.